IotWeb NET45/Helper/SessionFileStorageHandler.cs

```diff
@@ -0,0 +1,172 @@
+        private const string StorageFolder = "IoTSession";
+
+        public SessionFileStorageHandler(SessionStorageConfiguration sessionStorageConfiguration)
+        {
+            _sessionStorageConfiguration = sessionStorageConfiguration;
+        }
+
+        public SessionStorageConfiguration SessionStorageConfiguration
+        {
+            get
+            {
+                return _sessionStorageConfiguration;
+            }
+        }
+
+        public async Task<bool> DeleteSessionsAsync()
+        {
+            try
+            {
+                await Task.Run(() =>
+                {
+                    var path = GetStoragePath();
+
+                    if (File.Exists(path))
+                    {
+                        string[] files = Directory.GetFiles(GetStoragePath());
+
+                        foreach (string file in files)
+                        {
+                            FileInfo fi = new FileInfo(file);
+                            if (fi.LastAccessTime < DateTime.Now.AddMinutes(-SessionStorageConfiguration.SessionExpiredTime))
+                                fi.Delete();
+                        }
+                    }
+                });
+
+                return true;
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+        }
+
+        public async Task<bool> DeleteSessionAsync(string sessionId)
+        {
+            try
+            {
+                await Task.Run(() =>
+                {
+                    string[] files = Directory.GetFiles(GetStoragePath());
+
+                    foreach (string file in files)
+                    {
+                        if (file.Contains(sessionId))
+                        {
+                            FileInfo fi = new FileInfo(file);
+                            fi.Delete();
+                            break;
+                        }
+                    }
+                });
+
+                return true;
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+        }
+
+        public async Task<string> GetDataAsync(string fileName)
+        {
+            try
+            {
+                string data = null;
+
+                await Task.Run(() => {
+                    if (!string.IsNullOrWhiteSpace(fileName))
+                    {
+                        string fullFilePath = GetFilePath(fileName);
+
+                        if (File.Exists(fullFilePath))
+                        {
+                            data = File.ReadAllText(fullFilePath);
+                        }
+                    }
+                });
+
+                return data;
+            }
+            catch (Exception e)
+            {
+                return null;
+            }
+        }
+
+        public Task<Dictionary<SessionAttributes, object>> GetSessionMetadata(string fileName)
+        {
```

Comment:
Please implement it, Even if you are not using it, put a valid implementation here